src/output_manager.py

```diff
@@ -267,10 +268,22 @@ async def process_response(self, active_character: Character, blocking_queue: se
                                 if len(sentence.strip()) <= 3:
                                     logging.log(28, f'Skipping voiceline that is too short: {sentence}')
                                     break
-
+                                
                                 logging.log(self.loglevel, f"LLM returned sentence took {time.time() - start_time} seconds to execute")
                                 # Generate the audio and return the audio file path
                                 # Put the audio file path in the sentence_queue
+                                
+                                # Try to get the sentence below 148 characters which is the max for Fallout4
```

Comment:
I think the game engine can probably handle more, but I'm overwriting a CK-created record in memory, so I wanted to play it safe and not risk overwriting beyond the already existing record. 
Skyrim is doing things differently, so I don't think they have this limitation. I didn't think it would be a big limitation, though.